find-node example doesn't work on windows

Hey there, I'm having trouble running the example `find-node` step from the README.

I'm getting the following error in the logs for that step:


```
Run echo "node=$(node -e 'console.log(process.argv[0]);')" >> "$GitHub_OUTPUT"
  echo "node=$(node -e 'console.log(process.argv[0]);')" >> "$GitHub_OUTPUT"
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/cd[2](https://github.com/[redacted]/actions/runs/10149681769/job/28065132487#step:6:2)d0e4b-1e43-4e77-bc5b-3fd9849dbb59.sh: line 1: : No such file or directory
Error: Process completed with exit code 1.
```

Is it maybe an issue with the "$GITHUB_OUTPUT" variable on windows?

--- a/README.md
+++ b/README.md
@@ -48,7 +48,7 @@ sea-action:
       id: find-node
       run:
         echo "node=$(node -e 'console.log(process.argv[0]);')" >>
-        "$GitHub_OUTPUT"
+        $env:GITHUB_OUTPUT
 
     - name: SEA
       id: sea

